lib/polyfill/patchedmediakeys_apple.js

```diff
@@ -33,15 +34,15 @@ shaka.polyfill.PatchedMediaKeysApple = class {
       return;
     }
 
-    // TODO: Prefer unprefixed EME once we know how to use it.
-    // See: https://bugs.webkit.org/show_bug.cgi?id=197433
-    /*
+    // Only tested in Safari 14.
+    const safariVersion = shaka.util.Platform.safariVersion();
     if (navigator.requestMediaKeySystemAccess &&
-        MediaKeySystemAccess.prototype.getConfiguration) {
-      // Prefixed EME is preferable.
+        // eslint-disable-next-line no-restricted-syntax
+        MediaKeySystemAccess.prototype.getConfiguration &&
+        safariVersion && safariVersion >= 14) {
```

Comment:
I don't know how the whole patched media keys magic works in Shaka, but with `ts` HLS, you can only use prefixed EME, from my experience and understanding (let me know if I am wrong).

Doing so would prevent playing fairplay ts HLS on newer Safari versions, for developers using `shaka.polyfill.installAll()`. We could add a line in the documentation if this is wanted ?

Reply:
I just tested HLS in src = and it works fine on Safari 14, but not Safari 12. I don't have a Safari 13 to test.

Reply:
@valotvince, would you be able to test this in Safari 13?  Looking at Safari usage numbers, 13 is still much more common than 14, so it would be good to know.

Reply:
@valotvince Have you been able to test this?

Reply:
@avelad I'll take a look at it today or tomorrow

Reply:
Turns out Safari has been upgraded on my Mac so I only have Safari 14. I tested one of our content anyway, which is a HLS playlist with ts fragments, Fairplay-encrypted
- master: Playback is starting and decryption process works well 
- your branch: I have a `ENCRYPTED_CONTENT_WITHOUT_DRM_INFO` error

Reply:
Well, I have tried with different providers and streams, and of the 10 tests they have done, 9 have worked and 1 has failed. But I don't know what the reason is.

Reply:
Let's go ahead with this, as Safari 14 seems to be rolling out everywhere.  Please accept my apologies for our delays.